runelite-client/src/main/java/net/runelite/client/plugins/herbiboars/HerbiboarOverlay.java

```diff
@@ -41,6 +47,14 @@
 @Singleton
 class HerbiboarOverlay extends Overlay
 {
+	@Inject
+	@Getter
+	private Client client;
```

Comment:
Why does this have a getter? Also don’t inject it here but inject it in the constructor